app/mailers/payment_mailer.rb

```diff
@@ -0,0 +1,30 @@
+class PaymentMailer < Mail::Notify::Mailer
+  helper :application
+
+  def confirmation(payment, payment_date_timestamp)
+    @claim_description = claim_description(payment.claim)
+    @reference = payment.claim.reference
+    @payment = payment
+    @payment_date = Time.at(payment_date_timestamp).to_date
+
+    view_mail_with_payment_and_subject(payment, "We’re paying your #{@claim_description}, reference number: #{payment.claim.reference}")
+  end
+
+  private
+
+  def claim_description(claim)
+    I18n.t("#{claim.policy.routing_name.underscore}.claim_description")
+  end
+
+  def view_mail_with_payment_and_subject(payment, subject)
```

Comment:
Would it make the code easier to follow if the code in this method was simply inlined in the `confirmation` method? I get why this is here (it was present in the `ClaimMailer`), but over there it (kinda) justifies its existence as it DRYs up the setting of the display name and policy. I see fewer benefits where it is called once vs the indirection it creates.

Reply:
Agreed. I'll change it.